rero_ils/modules/loans/cli.py

```diff
@@ -394,8 +394,8 @@ def get_random_patron(exclude_this_barcode):
         .filter('term', organisation__pid=org_pid)\
         .source(['patron']).scan()
     for patron in patrons:
-        if patron.patron.barcode != exclude_this_barcode:
-            return Patron.get_patron_by_barcode(barcode=patron.patron.barcode)
+        if patron.patron.barcode[0] != exclude_this_barcode:
```

Comment:
since a patron can have multiple barcodes, this check is not valid anymore. Need to check that the `exclude_this_barcode` is not in the list `patron.patron.barcode`